src/mesh3D/Mesh3D.hpp

```diff
@@ -29,13 +29,16 @@ namespace polyfem
 		int n_edges() const override { return int(mesh_.edges.size()); }
 		int n_vertices() const override { return int(mesh_.points.cols()); }
 
-		inline int n_face_vertices(const int f_id) const { return mesh_.faces[f_id].vs.size(); }
-		inline int n_cell_vertices(const int c_id) const { return mesh_.elements[c_id].vs.size(); }
-		inline int n_cell_faces(const int c_id) const { return mesh_.elements[c_id].fs.size(); }
-		inline int cell_vertex(const int c_id, const int lv_id) const { return mesh_.elements[c_id].vs[lv_id]; }
-		inline int cell_face(const int c_id, const int lf_id) const { return mesh_.elements[c_id].fs[lf_id]; }
-		inline int cell_edge(const int c_id, const int le_id) const { return mesh_.elements[c_id].es[le_id]; }
-		inline int face_vertex(const int f_id, const int lv_id) const { return mesh_.faces[f_id].vs[lv_id]; }
+		inline int n_face_vertices(const int f_id) const {return mesh_.faces[f_id].vs.size(); }
+		inline int n_cell_vertices(const int c_id) const {return mesh_.elements[c_id].vs.size(); }
+		inline int n_cell_faces(const int c_id) const {return mesh_.elements[c_id].fs.size(); }
+		inline int cell_vertex(const int c_id, const int lv_id) const {return mesh_.elements[c_id].vs[lv_id]; }
+		virtual int cell_vertex_(const int f_id, const int lv_id) const override { return mesh_.elements[f_id].vs[lv_id]; }
+		inline int cell_face(const int c_id, const int lf_id) const {return mesh_.elements[c_id].fs[lf_id]; }
+		inline int cell_edge(const int c_id, const int le_id) const {return mesh_.elements[c_id].es[le_id]; }
+		inline int face_vertex(const int f_id, const int lv_id) const {return mesh_.faces[f_id].vs[lv_id]; }
```

Comment:
what changed here? indentation? and `cell_vertex_`?

Reply:
I don't understand why they all changed either. In my only one commit in this file, only one line is highlighted.[https://github.com/Huangzizhou/polyfem/commit/70b7592f806ece5f551bc56b81bebac1e485006f#diff-49485c042b6ce08288115da8d5885345261e4fd4268e0c8c612f746f6258ab98](url)

Reply:
I put a `_` at the end because there's a function which is not virtual and has the same name. Otherwise can I change the `cell_vertex` to virtual? (Sorry this is my first time using a virtual function)